test/specs/test.claim-compliance-feature-assurance.journeys.js

```diff
@@ -0,0 +1,92 @@
+import { browser, $, expect } from "@wdio/globals";
+import {
+  createSheepReviewClaim,
+  createSheepReviewForAdditionalHerd,
+  getDevSignInUrl,
+  enterPreMHReleaseDateAndContinue,
+} from "../utils/common.js";
+import { getClaimSelectorFromTable } from "../utils/backoffice-selectors.js";
+
+const assertAllClaimsAreInCheck = async (claimNumbers) => {
+  claimNumbers.forEach((claimNum) => expect(isClaimStatusInCheck(claimNum)).toBe(true));
+};
+
+const assertSomeClaimsAreOnHold = async (claimNumbers) => {
```

Comment:
Should we not be checking specifically which ones get moved to on hold? Rather than some

Reply:
Ah fair enough!